administrator/components/com_associations/views/association/tmpl/edit.php

```diff
@@ -235,13 +251,15 @@
 $extension  = $input->get('extension', '', 'string');
 $rLanguage  = $input->get('referencelanguage', '', 'string') != null ? $input->get('referencelanguage', '', 'string') : '';
 ?>
-<button id="toogle-left-panel" class="btn btn-small">Show/Hide Reference (PoC)</button>
+<button id="toogle-left-panel" class="btn btn-small"><?php echo JText::_('COM_ASSOCIATIONS_EDIT_HIDE_REFERENCE'); ?></button>
 
 <form action="<?php echo JRoute::_(
 			'index.php?option=com_associations&view=association&layout=' . $layout . '&acomponent='
 			. $aComponent . '&aview=' . $aView . '&extension=' . $extension . '&referencelanguage=' . $rLanguage . '&id='
 			. $this->referenceId
-		); ?>" method="post" name="adminForm" id="adminForm" class="form-validate" data-associatedview="<?php echo $this->associatedView; ?>">
+		); ?>" method="post" name="adminForm" id="adminForm" class="form-validate" data-associatedview="<?php echo $this->associatedView; ?>"
+		data-show-reference="<?php echo JText::_('COM_ASSOCIATIONS_EDIT_SHOW_REFERENCE'); ?>"
+		data-hide-reference="<?php echo JText::_('COM_ASSOCIATIONS_EDIT_HIDE_REFERENCE'); ?>">
```

Comment:
please nove the data attributes to the element that uses it, ie, the button itself.

data-* elements are always related to the html element itself.
So if this is text of the button it should be in the button html tag.